src/popup/components/Modals/PayloadDescription.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <Default
+    :icon="icon || 'info'"
+    v-bind="$attrs"
+    :title="$t('modals.payload-info.title')"
+    class="payload-description"
+    full-width
+  >
+    <template #msg>
+      <div class="msg first">
+        {{ $t('modals.payload-info.msg') }}
+      </div>
+      <div class="msg second">
+        {{ $t('modals.payload-info.desc') }}
+      </div>
+    </template>
+  </Default>
+</template>
+
+<script>
```

Comment:
I suggest use TS for new components to avoid too much work when migrating to Vue 3.